src/headless/HeadlessUtils.h

```diff
@@ -0,0 +1,25 @@
+/*
+** HeadlessUtils provides a collection of utility functions to manage and output
+** data and construct and manipulate surge
+*/
+#pragma once
+
+#include "SurgeSynthesizer.h"
+
+namespace Surge
+{
+namespace Headless
+{
+
+SurgeSynthesizer* createSurge(int sr);
+
+typedef std::vector<float> audioData_t;
+
+void writeWavFile(const audioData_t& data, int nChannels, std::string fname);
+void writeGnuplotFile(const audioData_t& data, int nChannels, std::string fname);
```

Comment:
Even if we know that these would be needed there is no really any good reason to add any dummy functions. When you don't have to keep a stable API/ABI things can be always refactored if they turn out to be not a good fit.

Reply:
Agree completely; in this case these were part of an ‘implied api’ for your consideration. If these functions existed would you use these signatures? Again this PR is more a proposal than a bit of code. (Although I have a draft of writeWavFile on mac kicking around)

Reply:
Probably yes. But we have another review process lets say for my gnuplot commits :) We will then look at the best alternative, which is probably the above.

Reply:
Yup cool so I will remove it. It was definitely one of those "could I imagine the API" things not "this is finished" things :)